docs/4.0/i18n/zh-Hans/Development-Guidelines/resource-controller.md

```diff
@@ -0,0 +1,119 @@
+# Resource-controller如何编写
+
+resource-controller 是计量计费模块的一部分，请确保您提前了解[计量计费的设计](https://github.com/labring/sealos/blob/main/proposal/design/zh/metering%E8%AE%BE%E8%AE%A1.md)
+
+
+
+## resource-controller 是什么
+
+resource-controller 是计量计费模块的一部分，用来统计和计费资源的使用量，不同资源不同resource-controller，使用podresource-controller举例，podresource-controller会统计用户ns下pod的资源limit使用量，并且计算出价格，生成一个Resource CR.
+
+![](https://p.ipic.vip/p87a3h.png)
```

Comment:
replace with repo url

Reply:
<img src="../../../../../proposal/img/metering-proposal-3.png" style="zoom:33%;" /> is ok?